wire/test/msgtest.go

```diff
@@ -12,29 +12,30 @@
 // See the License for the specific language governing permissions and
```

Comment:
Since "test" is now in the pathname, you can rename the file to just "msg.go"?

Reply:
See https://github.com/hyperledger-labs/go-perun/pull/287#discussion_r765649069